src/state/selectors.ts

```diff
@@ -48,6 +48,22 @@ export const getBuddyName = (
   );
 };
 
+export const getBuddyStatus = (
+  buddyId: string,
+  buddyState: AppState['buddies'],
+) => {
+  const look = (fa: RD.RemoteData<unknown, Record<string, { status: 'Banned' | 'Active' }>>) =>
+    RD.remoteData.map(fa, a => record.lookup(buddyId, a));
+  const buddy = getOptionM(RD.remoteData).alt(look(buddyState), () =>
+    look(buddyState),
+  );
```

Comment:
Lol. This is way too fancy. I think we can do this without monadtransformes https://en.wikibooks.org/wiki/Haskell/Monad_transformers :D?

Reply:
at least drop that alt, we don't have alternative place to look for ban status atm.

Reply:
Return type would be super nice for this also : )